### PR TITLE
Fix swagger specification for Alfresco MediaType

### PR DESCRIPTION
**Pull request by erdemedeiros**
Reuse common swagger configuration defined in common repository.

Refs https://github.com/Activiti/Activiti/issues/2588

### CODE REVIEW

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/api/ProjectRestApi.java [hunk 1]
```diff
@@ -86,8 +85,7 @@
             value = "List projects",
             notes = "Get the list of available projects. " +
                     "Minimal information for each project is returned.",
-            produces = APPLICATION_JSON_VALUE,
-            response = AlfrescoProjectsPage.class)
```
Comment (erdemedeiros) [inline activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/api/ProjectRestApi.java:90]: @constantin-ciobotaru I don't think this is necessary anymore, so I'm removing it. Let me know if it causes any issue.

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/swagger/SwaggerConfiguration.java [hunk 2]
```diff
@@ -16,60 +16,22 @@
...
 /**
  * Swagger configuration
  */
 @Configuration
-@EnableSwagger2
 public class SwaggerConfiguration {
 
-    @Bean
-    @ConditionalOnMissingBean(Docket.class)
-    public Docket api() {
```
Comment (erdemedeiros) [inline activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/swagger/SwaggerConfiguration.java:43]: I don't think this bean is ever used, as there is another one declared in the starter.

File: activiti-cloud-starter-org/src/main/java/org/activiti/cloud/organization/OrganizationSwaggerConfiguration.java [hunk 3]
```diff
@@ -16,27 +16,33 @@
...
-                .apis(RequestHandlerSelectors.basePackage("org.activiti.cloud.services.organization.rest"))
-                .paths(PathSelectors.any())
-                .build();
+    public Predicate<RequestHandler> apiSelector() {
+        return RequestHandlerSelectors.basePackage("org.activiti.cloud.services.organization.rest")::apply;
+    }
+
+    @Bean
+    public DocketCustomizer docketCustomizer(){
+        return (docket) -> docket.apiInfo(new ApiInfoBuilder()
```
Comment (constantin-ciobotaru) [inline activiti-cloud-starter-org/src/main/java/org/activiti/cloud/organization/OrganizationSwaggerConfiguration.java]: Can you make the apiInfo a bean (with @ConditionalOnMissingBean), to allow re-defining it in APS? I think, this should be the same for all swagger apiInfos created in Activiti.

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/swagger/SwaggerConfiguration.java [hunk 4]
```diff
@@ -16,60 +16,22 @@
...
-    @Bean
-    public ApiInfo apiInfo() {
-        return new ApiInfoBuilder()
-                .title("Activiti Modeling REST API")
-                .description("Provides access to the core features of Activiti Modeling.")
-                .version("1.0")
-                .build();
-    }
-
     @Bean
```
Comment (constantin-ciobotaru) [inline activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/swagger/SwaggerConfiguration.java]: I don't think we really need to define this bean. I think it was experimentally done for showing the call duration.

Comment (erdemedeiros) [reply activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/swagger/SwaggerConfiguration.java]: Ok. So I'll remove it.

File: activiti-cloud-starter-org/src/main/java/org/activiti/cloud/organization/OrganizationSwaggerConfiguration.java (hunk 3 above)
Comment (erdemedeiros) [reply activiti-cloud-starter-org/src/main/java/org/activiti/cloud/organization/OrganizationSwaggerConfiguration.java]: It makes sense. I'll update It.

**Review by constantin-ciobotaru — APPROVED** [review]

**Pull request merged**